accepted/2024/PMUI-Readme-rendering.md

```diff
@@ -22,18 +22,54 @@ The PM UI will be updated to have tabs for the Package Details and the the READM
 The README will be available in all the PM UI tabs.
 It will also be displayed for both the solution level and project level package managers.
 
+On the first time loading the PMUI we will default to the README tab, if the README tab isn't available then we will select the Package Details tab.
+The selected tab will be maintained between sessions and when switching packages.
+
 ![Alt text](../../meta/resources/ReadMePMUI/ReadmeTab.png)
 ![Alt text](../../meta/resources/ReadMePMUI/NoReadmeTab.png)
 
 #### README Tab
 This tab is rendered if a remote source available to download the readme or the Installed tab is selected and the selected version is found in the global packages folder.
 This is to ensure that users using the Browse tab will not see the README tab if they do not have a remote source that allows README downloads.
 
-If the README tab is rendered and we do not find a README then a message is rendered encouraging the user to contact the package author to upload a README.
-![alt text](../../meta/resources/ReadMePMUI/NoReadMeFound.png)
+When there is no README available we will display a message to the user in the README tab.
+>There is no README available for the selected package version. For more information please visit [aka.ms/nuget/noreadme](https://aka.ms/nuget/noreadme).
+
+The selected tab will not change even when the newly selected package does not have an embedded README. 
 
 If there is an error rendering the README then a message will be displayed for the users.  
 
+
+```mermaid
+---
+title: README tab visible?
+---
+flowchart TD
+    A(Which PM UI Tab Selected?)
+    A --> |Browse|B(Source supports downloading README?)
+    B --> |Yes|C(README tab visible)
+    B --> |No|D(README tab hidden)
+    A --> |Install/Update/Consolidate|E(Source supports downloading README?)
+    E --> |Yes|C
+    E --> |No|F(Selected package version is in GPF?)
+    F --> |Yes|C
```

Comment:
I'm still of the opinion that: 

Source supports downloading readme -> No _> Selected package version is in GPF? -> Yes -> Readme Tab visible 

should be

Source supports downloading readme -> No _> Selected package version is in GPF? -> Yes -> Does package have readme? -> Yes - Readme Tab Visible.

The default view should be what the package itself has. 

The feedback was the readme tab selection should be readme existence aware but it still isn't.